practices/managing-node-projects.md

```diff
@@ -18,54 +24,80 @@
 The following guide describes how we use node and manage package dependencies in [Springer Nature Node.js-based projects](https://github.com/springernature?utf8=%E2%9C%93&q=&type=public&language=javascript).
 
 
-## Specifying versions of node
+## Installing dependencies
 
-It's important to specify which versions of node your application expects. There are two ways of doing this, and you're encouraged to do both.
+There are two methods for installing node dependencies: `npm install` and `npm ci`.
```

Comment:
I disagree with your approach as there are several use cases;
 - "I just want to install the things" vs "i am seeking to actively develop this project"
 - apps vs libs
 - projects may or may not properly have a lockfile checked in

There are too many permutations to produce a prescriptive guide, and in any case I do not recall any principles of the runbook saying that we default to that approach.

"How to install node deps" seems a good place to start to teach these topics, rather than starting to talk about lockfiles, when lockfiles may or may not be relevant to my use case, and I may not even know what a lockfile is. For these reasons this doc is intended to teach concepts around `i` & `ci` and is more "educational" than most -- it is a complicated topic.